test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/SwaggerUITests.cs

```diff
@@ -137,5 +141,140 @@ public void Given_NullOptions_When_AddServer_Invoked_Then_It_Should_Return_Swagg
 
             (fi.GetValue(ui) as string).Should().Be(expected);
         }
+
+
+        [DataTestMethod]
+        [DataRow(OpenApiAuthLevelType.Anonymous, null, false, false)]
+        [DataRow(OpenApiAuthLevelType.Anonymous, "", false, false)]
+        [DataRow(OpenApiAuthLevelType.Anonymous, "configKey", false, false)]
+        [DataRow(OpenApiAuthLevelType.User, null, true, false)]
+        [DataRow(OpenApiAuthLevelType.User, "", true, false)]
+        [DataRow(OpenApiAuthLevelType.User, "configKey", false, true)]
+        [DataRow(OpenApiAuthLevelType.Function, null, true, false)]
+        [DataRow(OpenApiAuthLevelType.Function, "", true, false)]
+        [DataRow(OpenApiAuthLevelType.Function, "configKey", false, true)]
+        [DataRow(OpenApiAuthLevelType.System, null, true, false)]
+        [DataRow(OpenApiAuthLevelType.System, "", true, false)]
+        [DataRow(OpenApiAuthLevelType.System, "configKey", false, true)]
+        [DataRow(OpenApiAuthLevelType.Admin, null, true, false)]
+        [DataRow(OpenApiAuthLevelType.Admin, "", true, false)]
+        [DataRow(OpenApiAuthLevelType.Admin, "configKey", false, true)]
+        public async Task Given_Options_When_IsAuthKeyRequired_Invoked_Then_It_Should_Return_Result(
```

Comment:
Because this test method doesn't call any aync/await method, it should be `public void` instead of `public async Task`.

Reply:
oh thats true...i changed it